content/conditions/indigestion/main-content.md

```diff
@@ -0,0 +1,99 @@
+## How to tell if you have indigestion (dyspepsia)
+
+You can have the following symptoms after eating or drinking:
+
+- heartburn - a burning feeling in the chest, often after eating
+- feeling full and bloated
+- feeling sick
+- belching and farting
+- bringing up food or bitter tasting fluids
+
+
+## When it’s not indigestion
+
+Stomach ache or back pain are usually not symptoms of indigestion. If you have those you might be constipated. 
+
+
+## How you can treat indigestion yourself
+
+There’s usually no need to see a GP about indigestion. 
+
+Here are some things you can do at home:
+
+- avoid rich, spicy, fatty foods
+- cut down on tea, coffee, cola, alcohol
+- don’t eat 3 to 4 hours before going to bed 
+- prop your head and shoulders up in bed - this avoids acid from your stomach coming up while you’re sleeping
+- don’t take ibuprofen or aspirin - this can make indigestion worse 
+- lose weight if you’re overweight 
+- give up smoking
+
+These things will also help prevent indigestion. 
+
+
+## Your pharmacist can help with indigestion
+
+Your pharmacist can recommend medicines that’ll ease the burning feeling or pain that can come with indigestion. 
+
+These medicines make your stomach less acidic. They’re called antacids. 
+
+It’s best to take the medicine after eating. On a full stomach they’ll last up to 3 hours. On an empty stomach they’ll only last 20 minutes to an hour. 
+
+
+!!! info
+  [Find your nearest pharmacy](https://beta.nhs.uk/finders/find-help)
```

Comment:
Same thing here with using `{{ findersBaseUrl }}` instead of hard coding beta.nhs.uk